.doppler-ci

```diff
@@ -0,0 +1 @@
+Jenkinsfile
```

Comment:
I like this.
This file, should be added to `HelloMicroservice.sln`?

Reply:
Mmm, I think that it is not a good idea because it is only a symbolic link to Jenkinsfile. But I have no an strong opinion.

Thoughs?

Reply:
Could be not added.